packages/next/src/lib/metadata/resolve-metadata.ts

```diff
@@ -233,6 +237,10 @@ function merge({
       case 'metadataBase':
         target.metadataBase = metadataBase
         break
+      case 'themeColor': {
+        target.themeColor = resolveThemeColor(source.themeColor)
+        break
+      }
```

Comment:
why does themeColor and viewport show up here at all? for merging these concepts just aren't part of the API anymore right